src/redux/books/books.js

```diff
@@ -0,0 +1,21 @@
+const ADD = 'app/books/BOOK_ADDED';
+const REMOVE = 'app/books/BOOK_REMOVED';
+const reducer = (state = {}, action) => {
```

Comment:
- [ ] As per the requirements, you need to set the initial state to an array because it's ideal for storing multiple objects. But from your solution, you used an empty object literal. Please update this.

See example:
```js
const reducer = (state = [ ], action) => { 
     // your code goes here
};
```